src/components/Board.js

```diff
@@ -16,18 +24,97 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+    this.props.updateStatusCallback('Loading Cards...', 'success');
+
+    axios.get(CARDS_URL)
+    .then((response) => {
+      console.log('Success');
+      // console.log(response);
+
+      this.props.updateStatusCallback('Successfully loaded cards, Take one!', 'success');
+      // if we wanted only a first 10 cards, pre-processing on data, additional pre-processing or logic on data would be done here
+      const cards = response.data
+      // To get all of the cards, we reset the state with the response data (which is the cards)
+      this.setState({ cards: cards });
+    })
+    .catch((error) => {
+      // console.log('Error :(');
+      // console.log(error);
+      // Alert user to errors on screen by altering the status we are tracking with state
+      this.props.updateStatusCallback(error.message, 'error');
+    });
+  }
+
+  // FIXME: This portion of functionality is not working. I keep receiving a 404 Not Found, and need to turn it in.
+
+  // need a callback function that we will then pass to the NewCardForm
+  // Call back function for deleting card
+  deleteCard = ( index, id) => {
+    axios.delete(CARDS_URL + `${id}`)
+    .then((response) => {
```

Comment:
It looks like you're missing a `/` in your URL before the ID. When I added that, this code worked fine.

The way I found this was by noticing the error code - 404 indicates not found, which means either you have a bad card ID, or a bad URL format. I then looked in Chrome's network tab to see what you were actually sending, and noticed the missing `/`.